src/frontend/screens/MapScreen/MapLayers/RemoteDetectionAlertsLayer.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import MapboxGL from '@rnmapbox/maps';
+
+import {RemoteDetectionAlert} from '@comapeo/schema';
+import {FeatureCollection} from 'geojson';
+import {useRemoteDectionAlerts} from '../../../hooks/server/remoteDetectionAlert';
+import {flatten} from 'flat';
+import {includeKeys} from 'filter-obj';
+
+export const RemoteDectionAlertsMapLayer = () => {
```

Comment:
I suggest extracting any of the constant `filter` props used in this component to constants that live inside this file but outside the component

Reply:
```suggestion
export const RemoteDetectionAlertsMapLayer = () => {
```

Reply:
Im not sure I fully understand what you mean? Are you saying:

```ts
<MapboxGL.SymbolLayer
        id="mapeo-alerts-label"
       // Make the object inside "filter" a CONST?
        filter={[
          'all',
          [
            'in',
            '$type',
            'Polygon',
            'LineString',
            'Point',
            'MultiLineString',
            'MultiPolygon',
            'MultiPoint',
          ],
          ['has', 'metadata.alert_type'],
          ['has', 'month_detec'],
          ['has', 'year_detec'],
        ]}
```

Reply:
yeah - I'm suggesting defining those filters as consts that live outside the component as a minor optimization

Reply:
So it should look like:

```ts
// outside the component:
ALERTS_LABEL_FILTER = [
          'all',
          [
            'in',
            '$type',
            'Polygon',
            'LineString',
            'Point',
            'MultiLineString',
            'MultiPolygon',
            'MultiPoint',
          ],
          ['has', 'metadata.alert_type'],
          ['has', 'month_detec'],
          ['has', 'year_detec'],
        ]

// Inside the component:
<MapboxGL.SymbolLayer
        id="mapeo-alerts-label"
        filter={ALERTS_LABEL_FILTER}/>
        
```

Is that what you mean?

Reply:
https://github.com/digidem/comapeo-mobile/pull/895/commits/5506b1f3d00344d92637693a41f48137e48fd627